documentation-versioning.rst

```diff
@@ -0,0 +1,71 @@
+.. post:: July 23, 2019
+   :tags: versioncontrol, documentation
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Recommendations about how to do documentation versioning.
+
+Documentation Versioning: Best Practices
+========================================
+
+Project releases and documentation versions are strongly related each other.
+You usually want to have a release ``1.0.3`` with it's own documentation version ``1.0.3`` for that specific release.
```

Comment:
Django and Flask don't do this. I'd actually say if you're following semver you probably don't need to document below the minor version level (eg. `1.0.x`).